packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -3154,18 +3165,18 @@ export class StripeHelper {
     const subscription = await this.stripe.subscriptions.retrieve(
       (event.data.object as Stripe.Subscription).id
     );
-    // For PayPal customers, Stripe fails to send a `customer.updated` webhook
-    // when it automatically updates the `customer.currency` on subscription
-    // creation. Update the cached customer here to ensure `customer.currency`
-    // is not `null`.
-    // Since we can't easily differentiate a PayPal customer from others with
-    // an unexpanded subscription object, we update all customers in Firestore.
-    if (event.type === 'customer.subscription.created') {
```

Comment:
Why do we also want to do this for `customer.subscription.updated`? Is this to catch and fix the issue eagerly e.g. when `latest_invoice` changes on a renewal?

Reply:
Yep, its to catch the scenario earlier when the subscription hits its renewal period.